docs/source/sphinxext/gallery_generator.py

```diff
@@ -138,7 +138,9 @@ def build_gallery(srcdir, gallery):
     target_dir = os.path.join(srcdir, f"nb_{gallery}")
     image_dir = os.path.join(target_dir, "_images")
     source_dir = os.path.abspath(
-        os.path.join(os.path.dirname(os.path.dirname(srcdir)), "notebooks")
+        os.path.join(
+            os.path.dirname(os.path.dirname(srcdir)), "docs", "source", "pymc-examples", "notebooks"
+        )
```

Comment:
for now I've put the submodule in `docs/source`, later it could be taken out with (Michael had mentioned nbsphinx_link)